.github/workflows/ci.yml

```diff
@@ -36,13 +36,13 @@ jobs:
           host: ubuntu-latest
           cross: true
 
-        - target: x86_64-apple-ios
-          host: macos-latest
-          cross: true
+      # - target: x86_64-apple-ios
+      #   host: macos-latest
+      #   cross: true
 
-        - target: aarch64-apple-ios
-          host: macos-latest
-          cross: true
+      # - target: aarch64-apple-ios
+      #   host: macos-latest
+      #   cross: true
```

Comment:
Doing this will disable the builds altogether while we could just exclude the problematic `ipfs-http` on *these platforms*. Will comment suggestion below with the relevant build step.